backend/src/main/java/eu/bbmri_eric/negotiator/notification/NotificationController.java

```diff
@@ -28,4 +26,38 @@ public List<NotificationDTO> getNotificationsForUser() {
     return userNotificationService.getNotificationsForUser(
         AuthenticatedUserContext.getCurrentlyAuthenticatedUserInternalId());
   }
+
+  @GetMapping(
+      value = "/notifications/template/{templateName}",
+      produces = MediaType.APPLICATION_XHTML_XML_VALUE)
+  @Operation(
+      summary = "Get a notification email-template",
+      description = "Get a notification email-template by name")
+  @ResponseStatus(HttpStatus.OK)
+  public String getNotificationTemplate(@PathVariable String templateName) {
+    return userNotificationService.getNotificationTemplate(templateName);
+  }
+
+  @PostMapping(
+      value = "/notifications/template/{templateName}",
+      produces = MediaType.APPLICATION_XHTML_XML_VALUE)
+  @Operation(
+      summary = "Update a notification email-template",
+      description = "Update the content of a notification email-template by name")
+  @ResponseStatus(HttpStatus.OK)
+  public String updateNotificationTemplate(
+      @PathVariable String templateName, @RequestBody String template) {
+    return userNotificationService.updateNotificationTemplate(templateName, template);
+  }
+
+  @PostMapping(
+      value = "/notifications/template/{templateName}/reset",
+      produces = MediaType.APPLICATION_XHTML_XML_VALUE)
+  @Operation(
+      summary = "Reset a notification email-template",
+      description = "Reset a notification email-template to the default content")
```

Comment:
according to REST conventions we should not use verbs in path, so maybe figure out another way to achieve the same result?